tests/runtest.sh

```diff
@@ -653,6 +666,12 @@ function set_test_directories {
 function run_tests_in_directory {
     local testDir=$1
 
+    # Recursively search through directories for .sh files to prepare them.
+    for scriptFilePath in $(find "$testDir" -type f -iname '*.sh' | sort)
+    do
+        prep_test "${scriptFilePath:2}"
```

Comment:
@adityamandaleeka : I am batching the prep process for scripts here. As this change brings in dependencies between test shell scripts. So when one is executing it's dependencies might not have been prepped